### DISCUSSION TITLE
Welcome to Extras-Addon-for-YimMenu Discussions!

### DISCUSSION THREAD

**Discussion by Deadlineem** [Deadlineem/Extras-Addon-for-YimMenu#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by A5m0deu5** [comment]
Hey. I have some troubles using ur script: Once i'm injected YMM into GTA and press INSERT, two floating windows appears: "Money drop" and "Grief". If i click any place they'll disappear. Why these windows aren't inside "Extras addon" menu? I can't found this options inside of it also. If i reload script due settings in YMM, these windows appears again. Am I did something wrong? I'm using Alestarov menu, wasabi, minigame hack and Ultimate menu. After disabling all of these scripts my problem just saved even before game has been fully reloaded. I'm using Xenos Injector if this helps. Thank u.

**Comment by Deadlineem** [comment]
> Hey. I have some troubles using ur script: Once i'm injected YMM into GTA and press INSERT, two floating windows appears: "Money drop" and "Grief". If i click any place they'll disappear. Why these windows aren't inside "Extras addon" menu? I can't found this options inside of it also. If i reload script due settings in YMM, these windows appears again. Am I did something wrong? I'm using Alestarov menu, wasabi, minigame hack and Ultimate menu. After disabling all of these scripts my problem just saved even before game has been fully reloaded. I'm using Xenos Injector if this helps. Thank u.

These options are for the selected players tab, you will see them when you click on a players name in the menu, the reason they do this is a bug in YimMenu [https://github.com/YimMenu/YimMenu/discussions/2748](url)
to which they have not yet fixed.

Unfortunately, there is not a solution to get rid of them showing up on menu load/reload

**Extras has scripts from all of the scripts you have mentioned EXCEPT the minigame**, be sure to turn the rest of them off by going to Settings > Lua Scripts > click a script and click disable (or disable all and enable extras + minigame)

**Reply by A5m0deu5** [reply]
Oh, i didn't notice that this options appear in players tab, thank u! Yeah, i've read the note on the main page of the script so i've disabled everything except minigame :) 
Good wishes <3

**Reply by Deadlineem** [reply]
Yeah this was a recent change that we made, its still a little weird functioning but for the main player selectables, it make extras addon a bit less bloated.

**Reply by A5m0deu5** [reply]
Seems like that, btw i'm in ecstasy of ur job :p 
There is always some loopholes yeah? :D

**Reply by Deadlineem** [reply]
Haha thanks, the goal is to get everyone a good, legit, free menu that can do multiple things at once and not need to download 100+ different scripts to do it.  One script for everything!  Thanks for your support, Feel free to join us on discord you can join by adding my github name.

**Comment by darkwanderr** [comment]
Hey, I think it would be really nice if you could add a bodyguard spawner script to this. Yimmenu has this feature but it's really broken and it won't be fixed for a while, figured you could take up a challenge. When you spawn them they just follow you around and are not actual **bodyguards** for players, they attack NPCs when you shoot them but the same cannot be said for when you shoot real players. I would love to play with a crew in my car and gang on others and I'm sure some other people would share the same idea.

**Comment by Deadlineem** [comment]
> Hey, I think it would be really nice if you could add a bodyguard spawner script to this. Yimmenu has this feature but it's really broken and it won't be fixed for a while, figured you could take up a challenge. When you spawn them they just follow you around and are not actual **bodyguards** for players, they attack NPCs when you shoot them but the same cannot be said for when you shoot real players. I would love to play with a crew in my car and gang on others and I'm sure some other people would share the same idea.

YimMenu's bodyguard system does work, probably to the best of its ability?  Not sure though, ik it does attack players that have attacked you as well as NPC's but it seems to prioritize NPC's.  Have you submitted this to YimMenus github as an issue/improvement/bug?

**Reply by darkwanderr** [reply]
It actually doesn’t work that way, I’ve tested it beforehand. I had a bodyguard spawned next to me holding a pistol and when I was shot in the face by a player with shotgun he just stood there as if nothing ever happened. I commented about this bug in unknowncheats website under the menu post and I was told that it would not be fixed anytime soon due to its’ low priority compared to the many other bugs currently present in the menu itself. That’s why I asked you in the first place, to not wait for a single script for loong time.

**Reply by Deadlineem** [reply]
maybe the NPC expert @USBMenus can shed light on this.

**Comment by darkwanderr** [comment]
> maybe the NPC expert @USBMenus can shed light on this.

Oh well if it could be implemented maybe I could be of some assistance with this post of mine, it includes detailed information on how broken the script in yimmenu is.
https://github.com/YimMenu/YimMenu/issues/2854

**Reply by USBMenus** [reply]
I can look into it

**Comment by Morphfix** [comment]
So idk, how much my comment belongs here, its 7am and i didnt had any sleep. But through some random research i stumbled upon your Post and thought, that the Extra Addon really looks great. Just wanted to ask as a Question and for everyone to see, how good does the Menu work rn? Cuz i saw, that the Gifting Cars feature may not be working because of the new update. I know, i couldve just tried the addon myself, but im on my phone laying in bed and i thought it also may be good for people to know, how the menu looks like after the update. Hope this is the right place to ask
